Cannot find module sha.js when loading composer-common

<!--- Provide a general summary of the issue in the Title above -->

## Context
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->

```
$ npm install composer-common@next && node -r composer-common
npm WARN deprecated fs-promise@1.0.0: Use mz or fs-extra^3.0 with Promise Support

> protobufjs@6.6.3 postinstall /private/tmp/node_modules/protobufjs
> node scripts/postinstall

npm WARN saveError ENOENT: no such file or directory, open '/private/tmp/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/private/tmp/package.json'
npm WARN tmp No description
npm WARN tmp No repository field.
npm WARN tmp No README data
npm WARN tmp No license field.

+ composer-common@0.17.5
added 109 packages in 5.653s
module.js:538
    throw err;
    ^

Error: Cannot find module 'sha.js'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/private/tmp/node_modules/composer-common/lib/certificate.js:17:20)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
```

## Expected Behavior
<!--- Tell us what should happen -->
Should not get this error.

## Actual Behavior
<!--- Tell us what happens instead -->
Do get this error.

## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->
Give Simon penguin and add the missing dependency to package.json!

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug include code to reproduce, if relevant -->
1.
2.
3.
4.

## Existing issues
<!-- Have you searched for any existing issues or are their any similar issues that you've found? -->
- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/composer)

<!-- please include any links to issues here -->

## Context
<!--- How has this bug affected you? What were you trying to accomplish? -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used:
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

--- a/packages/composer-common/package.json
+++ b/packages/composer-common/package.json
@@ -77,6 +77,7 @@
     "proxyquire": "1.7.11",
     "rimraf": "2.5.4",
     "semver": "5.3.0",
+    "sha.js": "2.4.8",
     "sprintf-js": "1.0.3",
     "temp": "0.8.3",
     "thenify": "3.2.1",

